federation/pkg/federation-controller/configmap/configmap_controller_test.go

```diff
@@ -107,6 +112,7 @@ func TestConfigMapController(t *testing.T) {
 	// Test update federated configmap.
 	configmap1.Annotations = map[string]string{
 		"A": "B",
+		"federation.beta.kubernetes.io/cluster-selector": "{\"location\": \"europe\"}",
```

Comment:
Put `federation.beta.kubernetes.io/cluster-selector` as a constant in types.go.
Also I am still on the fence for calling it alpha or beta.